akka-http-cluster-management/src/test/java/akka/cluster/http/management/ClusterHttpManagementJavaCompileTest.java

```diff
@@ -0,0 +1,14 @@
+package akka.cluster.http.management;
+
+import akka.actor.ActorSystem;
+import akka.cluster.Cluster;
+
+public class ClusterHttpManagementJavaCompileTest {
```

Comment:
I think this will need an empty test method or Junit will freak out with "no test method found"
Just add `@Test public void compileOnly() {}`